src/lib/Helpers.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prefer-rest-params */
```

Comment:
@rmehri01 when you have time, can you get rid of this line and work on the errors?

Reply:
Sounds good!